README.md

```diff
@@ -1,24 +1,82 @@
-# README
+# Private Events
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+The private-event project allows the user to create and attend events that take place at a specific date and time.
 
-Things you may want to cover:
+## Build with
+* Ruby 3.0
+* Rais 6
+* SQL lite
 
-* Ruby version
+## Getting started
 
-* System dependencies
+To clone and run the application locally, use the following steps:
+- Clone the repository from GitHub, using the `git clone` command.
```

Comment:
Where do I need to run this command. 
Write the whole command that they need to run it correctly. 
`git clone https://github.com/gdumani/private-events.git`